### PR TITLE
Website: Add Sample Projects

### PR DESCRIPTION
**Pull request by cchrischen**
### Summary 
Add sample projects from this semester's trends class. Currently have 4/5 we asked to see how the components looks when populated with data. Images were compressed to be < 300kb.

### Notion/Figma Link 
[Notion](https://www.notion.so/Idol-FA24-1030ad723ce1804ca825d0ec8972c711?p=13b0ad723ce180f9a9fce10540ed5c63&pm=s)

### Test Plan 
see netlify deploy

### CODE REVIEW

**Comment by andrew032011** [comment]
Might need to workshop which screenshots we pick.

**Review by andrew032011 — APPROVED** [review]

**Review by Bookie0 — CHANGES_REQUESTED** [review]
can we remove the animation when the project accordion opens up?
like it slowly fades in, but i dont think we're having this animation show up anywhere else

File: new-dti-website/components/course/data/student_projects.json [hunk 1]
```diff
@@ -1,24 +1,29 @@
...
-      "title": "Project Name4",
-      "description": "This project is about blah blah blah blah blah blah blah blah blah blah blah blah blah.",
-      "imageSrc": "https://via.placeholder.com/400"
+      "title": "Off Campus Groups",
+      "description": "Centralized platform to find, join & create new events for OFF-campus events",
+      "imageSrc": "/course/projects/offcampusgroups.jpg"
+    },
+    {
+      "title": "CU Lost and Found",
+      "description": "A platform that helps students recover their lost belongings through crowdsourced data",
```
Comment (Bookie0) [inline new-dti-website/components/course/data/student_projects.json:25]:
```suggestion
      "description": "A platform that helps students recover their lost belongings through crowdsourced data.",
```

File: new-dti-website/components/course/data/student_projects.json [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
+      "title": "IKanji",
+      "description": "IKanji is a web app that enhances the learning of Japanese Kanji. It allows users to create custom lessons or use existing ones from the Genki Textbook that can be studied with IKanji's flashcards and multiple choice features. Additionally, it integrates AI to further enhance the educational experience.",
+      "imageSrc": "/course/projects/ikanji.jpg"
     },
     {
-      "title": "Project Name4",
-      "description": "This project is about blah blah blah blah blah blah blah blah blah blah blah blah blah.",
-      "imageSrc": "https://via.placeholder.com/400"
+      "title": "Off Campus Groups",
+      "description": "Centralized platform to find, join & create new events for OFF-campus events",
```
Comment (Bookie0) [inline new-dti-website/components/course/data/student_projects.json:20]:
```suggestion
      "description": "Centralized platform to find, join & create new events for OFF-campus events.",
```

**Review by Bookie0 — APPROVED** [review]
nice!

can we also [remove the red](https://www.notion.so/Idol-Lead-Sync-781de97ad403492b94b420349f829ea5?pvs=4#1550ad723ce1800fbbf4dbc5fe1fea35) on open state

Screenshot 2024-12-12 at 4 59 29 PM

File: new-dti-website/components/course/data/student_projects.json (hunk 1 above)
Comment (Bookie0) [reply new-dti-website/components/course/data/student_projects.json:25]: (adding period for consistency with the descriptions of the other projects)

**Pull request merged**